src/google/protobuf/compiler/cpp/helpers.cc

```diff
@@ -176,15 +176,15 @@ static const char* const kKeywordList[] = {
 #endif  // !PROTOBUF_FUTURE_BREAKING_CHANGES
 };
 
-static std::unordered_set<std::string>* MakeKeywordsMap() {
-  auto* result = new std::unordered_set<std::string>();
+static absl::flat_hash_set<std::string>* MakeKeywordsMap() {
+  auto* result = new absl::flat_hash_set<std::string>();
   for (const auto keyword : kKeywordList) {
     result->emplace(keyword);
   }
   return result;
 }
 
-static std::unordered_set<std::string>& kKeywords = *MakeKeywordsMap();
+static absl::flat_hash_set<std::string>& kKeywords = *MakeKeywordsMap();
```

Comment:
ugh, this is not safe to have at global scope

Reply:
ew good point, I'll just fix these now